packages/css-library/src/stylesheets/elements.scss

```diff
@@ -5,3 +5,158 @@
 // p {
 //    @include @usa-paragraph
 // }
+
+@use "../../dist/tokens/scss/variables" as *;
+
+$color-link-default-hover: rgba($color-black, 0.05);
+$small-screen:  481px;  // TODO: USWDSv3 breakpoints?
+$large-screen:  1201px; // TODO: USWDSv3 breakpoints?
```

Comment:
What is our plan with breakpoints? Are we adopting the USWDS v3 breakpoints? These variables are form Formation here:

https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/base/_b-breakpoints.scss

Reply:
As far as I know, we are replacing one breakpoint but keeping the others - https://vfs.atlassian.net/wiki/spaces/DST/pages/2731245815/DR001+VA+Design+System+v3+upgrade+rollout+steps

768px will be replaced with 640px.

Reply:
@micahchiang Do we have a ticket yet for adding breakpoints to the CSS Library? I guess at a minimum we need [these](https://design.va.gov/foundation/breakpoints#names-and-values) but I see [other stuff](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/base/_b-breakpoints.scss#L21) in Formation too that might need to be audited. And possibly some hidden things in these [imports](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/shared-variables.scss)?

Reply:
I'm just going to add the breakpoint tokens that I need for this PR for now: https://github.com/department-of-veterans-affairs/component-library/pull/982/commits/15b2b06fa98bdb3e6d27a44a9d8d941656570679

Reply:
Sorry, we don't have a ticket yet, I'll write one to audit and port the remaining stuff we need. I think I saw you added two in a recent commit.

Reply:
This is interesting - https://github.com/uswds/uswds/blob/62d297a7f09230e5fc4c8b9d9aaae099a4ebda2b/packages/uswds-core/src/styles/settings/_settings-utilities.scss#L34-L56 

I think we might be able to get away with just configuring what breakpoints we want to use in our [settings file](https://github.com/department-of-veterans-affairs/component-library/blob/8e80295be8a003232b1bae837f9e9fefb7f2ae31/packages/css-library/src/stylesheets/utilities.scss#L12). Question would remain about overriding some of the default pixel values though.

Reply:
eh, now that I think about it. We probably need to bring the breakpoints over anyway. There are plenty of places in vets-website where applications are using them directly in custom style sheets =\

Reply:
Ticket to add breakpoints to css library - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2334